### PR TITLE
Issue 7390 - Changing GetCommand to PostCommand for GetLog

### PR DESCRIPTION
**Pull request by mmerrell**
I made this change, but I'm not expecting it to get merged yet--I need to learn how to build & test the dotnet bindings. Mainly I want to see if the build passes in lieu of being able to run locally.

- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: dotnet/src/webdriver/Remote/RemoteWebDriver.cs [hunk 1]
```diff
@@ -172,7 +172,7 @@ public RemoteWebDriver(ICommandExecutor commandExecutor, ICapabilities desiredCa
                 // Only add the legacy log commands if the driver supports
                 // retrieving the logs via the extension end points.
                 this.CommandExecutor.CommandInfoRepository.TryAddCommand(DriverCommand.GetAvailableLogTypes, new CommandInfo(CommandInfo.GetCommand, "/session/{sessionId}/se/log/types"));
-                this.CommandExecutor.CommandInfoRepository.TryAddCommand(DriverCommand.GetLog, new CommandInfo(CommandInfo.GetCommand, "/session/{sessionId}/se/logs"));
+                this.CommandExecutor.CommandInfoRepository.TryAddCommand(DriverCommand.GetLog, new CommandInfo(CommandInfo.PostCommand, "/session/{sessionId}/se/logs"));
```
Comment (szamacz) [inline dotnet/src/webdriver/Remote/RemoteWebDriver.cs]: As per https://github.com/SeleniumHQ/selenium/issues/7390#issuecomment-518429009 also URL should be probably changed to one without trailing "s" ("/session/{sessionId}/se/log")

**Comment by mmerrell** [comment]
This was obviated by @jimevans :)

**Comment by giggio** [comment]
@mmerrell I noticed this was merged to `master`, but I think `master` will only end up on version 4, right? This should be backported to version 3 as well. I'm on `3.141.0` and the error occurs there too.

**Comment by mmerrell** [comment]
This PR wasn't merged to master--@jimevans made his own change, which replaced this. I've already closed out this PR and deleted the branch. If you're seeing issues in the code at this point, I'd recommend submitting a PR with whatever needs to be done to fix it. 
MM

**Comment by szamacz** [comment]
But @jimevans PR was merged do master as well and fix is only visible in version 4 as @giggio  mentioned. @mmerrell Can you make this change to be applied in version 3 as well, please?

**Comment by diemol** [comment]
@szamacz as @mmerrell said, this PR was not merged, the fix was done through another commit.

In addition to that, fixes are not ported to already relased versions. So the option is to use the upcoming release of the next Selenium 4 alpha. Just check the releases in the upcoming days.

**Comment by giggio** [comment]
If v4 was realeased I'd understand that, but it is still alpha, not even beta. What you are saying is that the users have to live with the bug on the stable version for maybe months, or use an unstable version. This is not cool. This has to be backported, as v3 also suffers from the problem.

**Comment by cgoldberg** [comment]
> This has to be backported

Nothing is stopping you from backporting the fix and building it yourself.

**Pull request closed without merging**